pennylane/templates/utils.py

```diff
@@ -80,10 +80,8 @@ def _get_shape(inpt):
     Raises:
         ValueError: if ``inpt`` fails to convert to a NumPy array
     """
-    try:
-        inpt = np.array(inpt)
-    except:
-        raise ValueError("Fails to convert {} to array.". format(inpt))
+    # Todo: Wrap this into a "try" statement
+    inpt = np.array(inpt)
```

Comment:
Just double checking the reason for this change and the todo?

Reply:
Because we could not test it, and the user has to be very inventive to raise an exception here. I would not even know which exception to expect.

But the input checks will get another round of grooming soon anyways.